manifests/package.pp

```diff
@@ -98,7 +98,7 @@
           content => template('msoffice/setup.ini.erb'),
           mode    => '0755',
           owner   => 'Administrator',
```

Comment:
Indeed it is, as I remarked in my opening comment. Thanks for digging up that custom fact! I wonder why I've never seen Puppet complain that one on any of my systems, though.